terraform/modules/aws-network-load-balancer/variables.tf

```diff
@@ -0,0 +1,27 @@
+variable "environment" {
+  type        = string
+  description = "name of the environment as a scope for the created resources (default: 'dev'; example: 'prod', 'staging')"
+  default     = "dev"
+}
+
+variable "node_port_http" {
+  type        = number
+  description = "HTTP port from the target machines that the LB forwards ingress on port 80 to"
+  default     = 8080
+}
+
+variable "node_port_https" {
+  type        = number
+  description = "HTTPS port from the target machines that the LB forwards ingress on port 443 to"
+  default     = 8443
+}
+
+variable "node_ips" {
+  type        = list(string)
+  description = "a list of private IPs from all nodes the load balancer forwards traffic to"
+}
+
+variable "subnet_ids" {
+  type        = list(string)
+  description = "a list of IDs from subnets where the nodes are connected to"
```

Comment:
is this the subnets of the instances, or the subnets the load balancer endpoints end up in?

Reply:
I adjusted the description. Let me know if that doesn't make it more clear. Some context as to why can be found [here](https://github.com/wireapp/wire-server-deploy/pull/299/files#diff-c1122f67096ade60b3228092a78720c7R34-R37)